src/app/rundown/components/rundown/rundown.component.html

```diff
@@ -1,21 +1,28 @@
 <div class="title">{{rundown?.name}}</div>
-<div class="container">
-  <div>
+
+<div *ngIf="isLoading; else loadedRundown" class='c-loading-rundown'>
+  <mat-card-title>Loading the rundown...</mat-card-title>
+</div>
+
+<ng-template #loadedRundown>
+    <div class="container">
     <div>
-      <button *ngIf="!rundown?.isActive; else activeRundownButtons" (click)="activateRundown()">Activate Rundown</button>
-      <ng-template #activeRundownButtons>
-        <button (click)="takeNext()">Take Next</button>
-        <button (click)="resetRundown()">Reset Rundown</button>
-        <button (click)="deactivateRundown()">Deactivate Rundown</button>
-      </ng-template>
-    </div>
-    <div class="segments">
-      <sofie-segment
-        *ngFor="let segment of rundown?.segments"
-        [segment]="segment"
-        [isOnAir]="segment.isOnAir"
-      ></sofie-segment>
+      <div>
+        <button *ngIf="!rundown?.isActive; else activeRundownButtons" (click)="activateRundown()">Activate Rundown</button>
+        <ng-template #activeRundownButtons>
+          <button (click)="takeNext()">Take Next</button>
+          <button (click)="resetRundown()">Reset Rundown</button>
+          <button (click)="deactivateRundown()">Deactivate Rundown</button>
+        </ng-template>
```

Comment:
If you want, this ng-template can be placed on the "top level":
```
// Your loading div

// Your #rundown template

// Your #activeRundownButtons template
```

I think it comes down to personal preference, but one of the benefits of templates is you can "extract" it from the surrounding markup just like you can extract a method.

Reply:
I extracted the template. I liked the idea of extracting it, and since it is only temporary anyway, i thought it best.